examples/README.md

```diff
@@ -44,18 +96,55 @@ examples
+
+If you need to add a new example or add to an existing example, please follow the [Folder Structure](#folder-structure) guidelines above.
+
+## Add to an existing example app
+
+The process may differ depending on how the example app is setup, but the recommendations we have made around creating a new example should follow the same structure as we've implemented for the [[Passport Connect with NextJS](/examples/passport/wallets-connect-with-nextjs) and [Passport Signing with NextJS](/examples/passport/wallets-signing-with-nextjs) examples.
+
+So if you need to add a new example to an existing example app you should create a branch in `ts-immutable-sdk` to add your example to and follow these steps;
+
+### Setup example in existing app
+
+1. Create new route folder inside the example app and add the `page.tsx` file.
+1. Add the new route you created to the app's home page
+1. Add any packages you require
+1. Add any environment variables your example requires to the `.env.example` file
+1. Add instructions in the `README.md` file about how to populate the `.env` file and any other build instructions.
+
+### Add your example
+
+Once you've done this you can go ahead and write your example code in the `page.tsx` file.
+
+You can run the example locally to test your code by following the steps in the [Running examples locally](#running-examples-locally) section.
+
+You will also need to create e2e tests for your example as covered in the [End to end testing](#end-to-end-testing) section.
+
+Creating code snippets in the docs site using your example code is covered in the [Using code examples in the docs site](#using-code-examples-in-the-docs-site) section.
+
+If your code example is going to be used as code snippets in the docs site, before merging your branch to main in `ts-immutable-sdk` you should follow the steps in [Using code examples in the docs site](#using-code-examples-in-the-docs-site). These steps help you to test the code snippets in the docs site before you merge your PR which avoids double handling and multiple pull requests across both repos.
+
+## Creating a new example app
+
+The process may differ if you're using a different Javascript Framework, but this assumes you will be creating a new example app using NextJS. These steps which were implemented to create the [Passport Connect](/examples/passport/wallets-connect-with-nextjs) example. It is a good reference point in terms of route structure and page layouts. If you need to copy code from there to help with your layouts then please go ahead.
 
-## Creating a New Example
+### Setup example app
 
-create the nextjs project
+If you want to make a new NextJS app from scratch this is the process;
 
+Create a branch in `ts-immutable-sdk` to add your example to.
+
+In the console navigate to the product directory where your example will live (or create one if it doesn't exist). Then use `yarn dlx` to create the app.
 ```
```

Comment:
Can we add syntax highlighting to all the codeblocks? For eg here we could do ```bash, and below in the code snippets for the config files for eg we could add ```ts/```js